build.proj

```diff
@@ -23,14 +23,21 @@
              Directories="obj" />
     <Exec Command="git rev-list --count HEAD" 
           ConsoleToMSBuild="true">
-      <Output TaskParameter="ConsoleOutput" PropertyName="CommitCount" />
+      <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitCount" />
     </Exec>
 
+    <ItemGroup>
+      <GitInfoCommitCountLines Include="$(GitInfoCommitCount)" />
+    </ItemGroup>
```

Comment:
I had the same question, I used only one variable without the extra itemgroup. It just won't work. I get 00000 as result. After 3 round i ended up adding all the ceremonies in CLI repo.